mcos/error_estimator/expected_outcome.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+from mcos.error_estimator import AbstractErrorEstimator
+
+
+class ExpectedOutcomeErrorEstimator(AbstractErrorEstimator):
+    """Error Estimator that calculates the mean difference in expected outcomes"""
+
+    def estimate(self, allocation: np.array, optimal_allocation: np.array) -> float:
+        return (optimal_allocation - allocation).mean()
```

Comment:
This method implements the formula as defined by the paper:
<img width="541" alt="Screen Shot 2020-01-27 at 8 35 41 PM" src="https://user-images.githubusercontent.com/24689747/73228470-aa5ca000-4144-11ea-880b-34ba721a5331.png">
where the first value is the the optimal allocation from the original mu/cov pair and the second value is the allocation from a simulated mu/cov pair.

Reply:
You have the (w - what), but you're missing the mu part. The original mu has to be part of the equation. It should be `np.dot(optimal_allocation - allocation, mu)`

Reply:
Updated this